src/servers/graphql-server.ts

```diff
@@ -224,9 +236,11 @@ export const startApolloServer = async ({
     }),
   )
 
+  const secret = jwksRsa.expressJwtSecret(getJwksArgs()) as GetVerificationKey // https://github.com/auth0/express-jwt/issues/288#issuecomment-1122524366
```

Comment:
will this invalidate current jwt tokens?

Reply:
not in the short term, but in the long term yes (the idea is to no longer use long term symmetric JWT)

let me update the PR to give more context on the approach to update the auth system